src/App.css

```diff
@@ -782,6 +782,12 @@ body.dark {
   z-index: 11; /* above breadcrumbs, toolbar, export modal and status */
 }
 
+.alert-inline {
+  width: auto;
+  left: 50%;
+  transform: translateX(-50%) !important;
```

Comment:
`!important` is considered a bad practice. You can increase the specificity of the selector if you need it to override another selector. Then at least it can be overridden in the future.

Reply:
The problem is, there is this useSwipeToDismiss hook that provides an inline style of `transform: translateY(0px)` that doesn't allow the inline alert style to be properly centered. So I thought marking the css property important will be the solution. So in this case I am not sure on how to increase the specificity of the selector. May be I could override that with an inline style or is it non recommendable as well?

Reply:
That's fair. Try an inline style. It is preferable since it can be overridden. If that doesn't work, add a descriptive comment to the `!important` line so that if we need to change it in the future we understand the implications.